.gitignore

```diff
@@ -22,6 +22,7 @@ downloads/
 eggs/
 .eggs/
 lib/
```

Comment:
I didn't feel confident removing this from the `.gitignore` so I just included another rule below to include our testing `/lib` but let me know!